src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2416,7 +2416,10 @@ namespace wallet_rpc
   {
     struct request_t
     {
+      bool enable_multisig;
```

Comment:
```suggestion
      bool enable_multisig_experimental;
```
For any user-facing API, add the experimental qualifier.